dotnet/src/Microsoft.Agents.AI.Hosting.OpenAI/Responses/AIAgentResponsesProcessor.cs

```diff
@@ -85,100 +54,20 @@ public Task ExecuteAsync(HttpContext httpContext)
             response.Headers.ContentEncoding = "identity";
             httpContext.Features.GetRequiredFeature<IHttpResponseBodyFeature>().DisableBuffering();
 
+            var chatMessages = createResponse.Input.GetInputMessages().Select(i => i.ToChatMessage()).ToList();
+            var events = agent.RunStreamingAsync(chatMessages, cancellationToken: cancellationToken)
+                .ToStreamingResponseAsync(createResponse, context, cancellationToken)
```

Comment:
out of curiousity, why this extension method lives in a separate class? it is used only here, so why not implement it in-place?